test/specs/contextpopup/ContextPopupDirective.spec.js

```diff
@@ -16,6 +16,12 @@ describe('ga_contextpopup_directive', function() {
       $provide.value('gaNetworkStatus', {
         offline: true
       });
+
+      $provide.value('gaLang', {
+        get: function() {
+          return 'de';
+        }
+      });
```

Comment:
Sorry I didn't see you mention the ContextPopup test.  Because here we replace the gaLang service by an object. So the real gaLang service is not loaded.